tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/Qualification.scala

```diff
@@ -43,12 +44,18 @@ class Qualification(outputDir: String, numRows: Int, hadoopConf: Configuration,
   logInfo(s"Threadpool size is $nThreads")
   private val threadPool = Executors.newFixedThreadPool(nThreads, threadFactory)
     .asInstanceOf[ThreadPoolExecutor]
-
+  private var progressBar: Option[ConsoleProgressBar] = None
   private class QualifyThread(path: EventLogInfo) extends Runnable {
     def run: Unit = qualifyApp(path, hadoopConf)
   }
 
   def qualifyApps(allPaths: Seq[EventLogInfo]): Seq[QualificationSummaryInfo] = {
+    progressBar =
+      if (enablePB && allPaths.length > 0) { // total count to start the PB cannot be 0
+        Some(new ConsoleProgressBar("Qual Tool", allPaths.length))
+      } else {
```

Comment:
Can we remove `else` since progressBar is initialized to None? Please let me know if I am missing something here.